snuba/datasets/processors/generic_metrics_processor.py

```diff
@@ -141,7 +141,11 @@ def process_message(
                 GRANULARITY_ONE_DAY,
             ],
         }
-        return InsertBatch([processed], None)
+        return InsertBatch(
+            [processed],
+            origin_timestamp=None,
+            sentry_received_timestamp=metadata.headers["sentry_received_timestamp"],
```

Comment:
so since we aren't doing the headers anymore, we can change this back to getting the value from the message